app/node_modules/dimensions/clientcommandhandler.ts

```diff
@@ -104,7 +104,11 @@ export class ClientCommandHandler {
     let dimensionsList: string = "";
     let dimensionNames: string[] = _.keys(client.servers);
     for (let i: number = 0; i < dimensionNames.length; i++) {
-      dimensionsList += (i > 0 ? "[c/00B530:,] " : " ") + "/" + dimensionNames[i];
+      let name: string = dimensionNames[i];
+      let hidden: boolean = client.servers[name].hidden;
+	    if (!hidden) {
```

Comment:
Indentation here looks wrong

Reply:
Well i tried building it and worked well, what is wrong with indentation exactly?

Reply:
Can't you see it in the preview above my comment? The `if` is indented more than the previous line when it shouldn't be.

Reply:
I see, it should be fixed in commit 0d74825d3432e85957d820156b1b8ca7d27c1e85